composer.json

```diff
@@ -22,7 +22,8 @@
     "zendframework/zend-diactoros": "~1.1",
```

Comment:
I'd be actually very happy to set min PHP version to PHP7. This would allows us to remove dependency to Zend-Math and performs various simplifications.